client/components/public/Home.jsx

```diff
@@ -29,12 +29,12 @@ const AnnouncementsMessage = (
                       <ul>
                         <li>Date: { eventDay }, { eventDate } from 9:30 AM&mdash;5:00 PM (PT).</li>
                         <li>Event is FREE and open to all (donations gratefully accepted).</li>
-                        <li>We hope to run a HYBRID event this year (virtual and in-person options).</li>
+                        <li>We plan to run a HYBRID event this year: In-Person (WWU campus community only this year) and Virtual options.</li>
                         <li>Due to COVID-19 and our Priority on Safety and Health, we will follow University and CDC Guidelines.</li>
                         <li>If necessary, we will adapt the event to be fully virtual.</li>
                       </ul>
                       <br />
-                      <center><strong>Check back in November for updates!</strong></center>
+                      <center><strong>Check back here for updates!</strong></center>
```

Comment:
Could we add a note with today's date saying we're delaying registration by a little longer to implement some technical upgrades?

Reply:
Will do! After all we did say there would be updates.